src/rust-wasm-template/README.md

```diff
@@ -0,0 +1,63 @@
+# rust-wasm-template
+
+This repository contains the minimum amount of code needed to experiment with WebAssembly. Including a web server to serve the HTML and WASM as well as the javascript boilerplate needed to load WASM.
+
+- `/static` contains the static files including compiled webassembly.
+- `/src/lib.rs` contains the Rust code.
```

Comment:
These paths are not relative to this directory (I'm assuming), so they should not start with `/`:

```suggestion
- `static/` contains the static files including compiled webassembly.
- `src/lib.rs` contains the Rust code.
```